homeassistant/components/device_tracker/__init__.py

```diff
@@ -226,6 +214,8 @@ def async_see_service(call):
     hass.services.async_register(
         DOMAIN, SERVICE_SEE, async_see_service, descriptions.get(SERVICE_SEE))
 
+    # restore
+    yield from tracker.async_setup_tracked_device()
```

Comment:
I know it with last_time. With last time it will be allready on state machine. The reasin is describe on topic. At the moment block a setup they can (i.e. all beacons) take very long.